components/picker/stories/template.js

```diff
@@ -39,6 +40,7 @@ export const Picker = ({
 				["is-open"]: isOpen,
 				["is-loading"]: isLoading,
 				["is-keyboardFocused"]: isKeyboardFocused,
+				[`${rootClass}--menu-height-fixed`]: menuHeight,
```

Comment:
Have we considered the pros and cons of not having a dedicated API for this feature? For instance, instead of having a specific menu-height class, we could use a generic mod variable applied to the base class. This mod variable would function as a feature toggle, although customers would need to define its value themselves (because we would not have a default fallback).

Reply:
But I guess this is a more holistic API design question, maybe the CSS team can help understand when a `mod` is enough vs a dedicated toggleable class feature.

Reply:
I agree, this feels like a good use-case to add a mod rather than new API.